src/widget/wvumeter.cpp

```diff
@@ -9,15 +9,35 @@
 #include "moc_wvumeter.cpp"
 #include "util/math.h"
 #include "util/timer.h"
+#include "waveform/sharedglcontext.h"
+#include "waveform/vsyncthread.h"
 #include "widget/wpixmapstore.h"
 
 #define DEFAULT_FALLTIME 20
 #define DEFAULT_FALLSTEP 1
 #define DEFAULT_HOLDTIME 400
 #define DEFAULT_HOLDSIZE 5
 
+namespace {
+QColor findBaseColor(QWidget* pWidget) {
+    int i = 0;
+
+    while (pWidget) {
+        if (pWidget->palette().isBrushSet(QPalette::Normal, QPalette::Base)) {
+            return pWidget->palette().color(QPalette::Base);
+        }
+        i++;
+        pWidget = qobject_cast<QWidget*>(pWidget->parent());
+    }
+    return QColor(0, 0, 0);
+}
+} // namespace
```

Comment:
This function could go to [/src/util/widgethelper.cpp](`https://github.com/mixxxdj/mixxx/blob/2.3/src/util/widgethelper.cpp`) so it's available for others as well

Reply:
will do